app-modules/report/src/Filament/Widgets/ServiceRequestsOverTimeLineChart.php

```diff
@@ -44,13 +44,9 @@ class ServiceRequestsOverTimeLineChart extends ChartReportWidget
 {
     protected static ?string $heading = 'Requests Over Time';
 
-    protected static ?string $maxHeight = '250px';
+    protected static ?string $maxHeight = '200px';
```

Comment:
Let's copy what we did for linecharts in AdvisingApp here.

In AdvisingApp we created an abstract class specifically for LineCharts that applied the max height and set the type. Then all LineCharts in the `report` module extended it.